### PR TITLE
New package: xplr-0.20.0

### PR DESCRIPTION
**Pull request by devraza**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/xplr/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Muhammad Nauman Raza <thedevraza@proton.me>"
+license="MIT"
+homepage="https://github.com/sayanarjit/xplr"
+distfiles="https://github.com/sayanarijit/xplr/archive/refs/tags/v${version}.tar.gz"
+checksum=c4d63d9e1e313eeeb2e6d8d17e30b18ee4b8be01c419f08a89959fe5a4a09ac0
+
+do_install() {
+	     vlicense LICENSE
+	     vdoc README.md
+	     vmkdir usr/share/doc/xplr/
```
Comment (classabbyamp) [inline srcpkgs/xplr/template:16]: this dir should already have been created by vdoc

File: srcpkgs/xplr/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+revision=1
+build_style=cargo
+short_desc="Hackable, minimal, fast TUI file explorer"
+maintainer="Muhammad Nauman Raza <thedevraza@proton.me>"
+license="MIT"
+homepage="https://github.com/sayanarjit/xplr"
+distfiles="https://github.com/sayanarijit/xplr/archive/refs/tags/v${version}.tar.gz"
+checksum=c4d63d9e1e313eeeb2e6d8d17e30b18ee4b8be01c419f08a89959fe5a4a09ac0
+
+do_install() {
```
[the remark this answers is not in the document]
Comment (devraza) [reply srcpkgs/xplr/template]: Updated to use post_install() - vbin has also been removed too.

File: srcpkgs/xplr/template (hunk 1 above)
Comment (devraza) [reply srcpkgs/xplr/template]: Removed creation of the ;usr/share/doc/xplr' directory.

**Comment by icp1994** [comment]
Check out the [Alpine patch](https://git.alpinelinux.org/aports/tree/testing/xplr/remove-vendored-lua.patch) for fixing the cross build failures. And squash all your commits into a single one as per the [contributing guidelines](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes).

**Pull request closed without merging**